src/ducks/categorization/services.js

```diff
@@ -22,6 +22,9 @@ const FAKE_TRANSACTION = {
   label: 'thisisafaketransaction',
   manualCategoryId: '0'
 }
+const LOCAL_MODEL_CATEG_FALLBACK = '0'
```

Comment:
Why is it a string 0 ?

Reply:
The category `'0'` means `uncategorized` in our data model. See [here](https://github.com/cozy/cozy-banks/blob/master/src/ducks/categories/tree.json)

Reply:
Can you add a comment ? Or better (IMHO), a variable `const UNCATEGORIZED_CAT_ID = '0'`. I feel like in 2 weeks, we won't remember this otherwise :).